eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/Mary/Examples.hs

```diff
@@ -16,23 +16,24 @@ import Cardano.Ledger.UTxO (UTxO)
 import Control.State.Transition.Extended hiding (Assertion)
 import Control.State.Transition.Trace (checkTrace, (.-), (.->>))
 import Data.Default.Class (def)
+import Data.List.NonEmpty (NonEmpty)
 import GHC.Stack
 import Lens.Micro
 import Test.Cardano.Ledger.Mary.TreeDiff ()
 import Test.Cardano.Ledger.Shelley.Utils (applySTSTest, runShelleyBase)
 import Test.Tasty.HUnit (Assertion, (@?=))
 
 ignoreAllButUTxO ::
-  Either [PredicateFailure (ShelleyLEDGER Mary)] (LedgerState Mary) ->
-  Either [PredicateFailure (ShelleyLEDGER Mary)] (UTxO Mary)
+  Either (NonEmpty (PredicateFailure (ShelleyLEDGER Mary))) (LedgerState Mary) ->
+  Either (NonEmpty (PredicateFailure (ShelleyLEDGER Mary))) (UTxO Mary)
```

Comment:
This is an interesting one with respect to versioning. This is not a user facing change, since it is in the test suite. But, nevertheless it is a change and if we don't bump up the version this package will not be released, which will lead to CHaPs failing CI (due to it trying to build an older version that is not compatible with newer small-steps)

So, this requires a minor version bump. But, because it has already been bumped recently you don't need to do anything about it in this PR :)

This one also I bring up as a good example of our versioning processes.